Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,15 @@
+class Pacapt < Formula
+
+  homepage "https://github.com/icy/pacapt" 
+  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4", :tag => "v2.0.2"
```

Comment:
Can we not use the tarball? If not, split the ` revision ` onto a separate line, otherwise the formula gets massively wide.